chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1641,7 +1642,7 @@ type NormalFilter = EverythingBut<(IsRelayRegister, IsParathreadRegistrar)>;
 impl pallet_maintenance_mode::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
     type NormalCallFilter = NormalFilter;
-    type MaintenanceCallFilter = MaintenanceFilter;
+    type MaintenanceCallFilter = InsideBoth<MaintenanceFilter, NormalFilter>;
```

Comment:
This is so that disabled pallets in `NormalFilter` are automatically disabled in maintenance mode even if we forget to add them to `MaintenanceFilter`. Without it, not sure if the starlight maintenance filter is correct.